examples/hitl/rearrange_v2/ui.py

```diff
@@ -1019,6 +1063,36 @@ def _draw_selected_object_highlights(self) -> None:
             for link_id in link_object_ids.keys():
                 object_ids.add(link_id)
 
+            # Draw an outline around the default receptacle.
+            # TODO: Cache the default receptacle.
+            if self._ui_settings.highlight_default_receptacles:
+                link_index = self._world.get_link_index(object_id)
+                default_link_index = sutils.get_ao_default_link(
+                    ao, compute_if_not_found=True
+                )
+                if (
+                    default_link_index is not None
+                    and default_link_index != link_index
+                ):
+                    default_link_node = obj.get_link_scene_node(
+                        default_link_index
+                    )
+                    if (
+                        default_link_node.object_semantic_id
+                        not in self._world._opened_link_set
+                    ):
+                        color_default_link = self._to_color_array(
+                            COLOR_HIGHLIGHT
```

Comment:
I think ok for now, but maybe in the future, should be a config variable?

Reply:
Agreed!
I introduced UI configuration in this PR. I intend to make it fully configurable in the following weeks.